### PR TITLE
Polished paragraph as per discussion

### PR DESCRIPTION
**Pull request by drorata**
@Kacawi: please have a look

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Author: Dror Atariah
 
-As part of the never-ending effort to improve [reBuy](https://www.rebuy.de/) and turn it into a market leader, we recently decided to tackle (some of) the challenges of our customer services agents using data: our plan was to craft a system that analyze the content of emails from our customers and, when applicable, reply automatically to some of them. 
+As part of the never-ending effort to improve [reBuy](https://www.rebuy.de/) and turn it into a market leader, we recently decided to tackle (some of) the challenges of our customer services agents using data: our plan was to craft a system that analyze the content of emails from our customers and, when applicable, reply automatically to some of them.
```
Comment (Kacawi-zz) [inline README.md]: Should be: "... a system that analyzes"

File: README.md [hunk 2]
```diff
@@ -393,8 +393,14 @@ terraform apply -var 'workersNum=2' -var 'instanceType="t2.small"' \
 **Note** the use of two environment variables for the AWS keys. Other variables defined in `var.tf` are passed as parameters.
 
 <!-- could you restructure this paragraph a bit? --->
-Once finished, you can access the newly created scheduler node by: `ssh -i ~/.aws/key.pem ec2-user@$(terraform output scheduler-info)`. For this to work, you might need to download a `pem` file from AWS console. In the cluster you can check the log at `/var/log/user-data.log`. You can also check the status of the running Docker containers using `docker ps`. Lastly, if everything went well, you should be able to access the web interface of the cluster.
-Its address can be found by invoking `terraform output scheduler-status`.
+Once finished, you can access the newly created scheduler node by:
```
Comment (Kacawi-zz) [inline README.md]: "by executing the following line of bash code?"

File: README.md [hunk 3]
```diff
@@ -393,8 +393,14 @@ terraform apply -var 'workersNum=2' -var 'instanceType="t2.small"' \
...
+
+```bash
+ssh -i ~/.aws/key.pem ec2-user@$(terraform output scheduler-info)
+````
+
+For this to work, you will need to download a `pem` file from AWS console. Once logged in to the scheduler, you can check the log, as defined in the provisioning script, at `/var/log/user-data.log`. You can also check the status of the running docker containers using `docker ps`. Lastly, if everything went well, you should be able to access the web interface of the `dask` cluster and verify that all workers are detected and "linked" to the scheduler.
```
Comment (Kacawi-zz) [inline README.md]: the web interface of the dask cluster and the dashboard you refer to in 403 are the same?

File: README.md [hunk 4]
```diff
@@ -460,4 +466,4 @@ A side story in this tutorial is the beauty of a declarative approach. If you ma
 There are two next steps that I haven't fully explored yet, but I believe they are definitely worth investigating further:
 
 * You might want to check out `terraform workspace`; this can help you run several clusters from the same directory. For example when running different experiements at the same time.
-* Enable a node with Jupyter server so the local notebook won't be needed
\ No newline at end of file
+* Enable a node with Jupyter server so the local notebook won't be needed
```
Comment (Kacawi-zz) [inline README.md]: "Enable a node with Jupyter server so that you won't need the local notebook"?

**Review by Kacawi-zz — COMMENTED** [review]
Hi - I had made some comments yesterday but forgot to submit the review, sorry! Will get back to your e-mail immediately also to explain the next steps!

**Comment by drorata** [comment]
@Kacawi Can you double check the updates I applied?

**Review by Kacawi-zz — APPROVED** [review]

**Comment by Kacawi-zz** [comment]
@drorata LGTM!

**Pull request merged**